pkg/controller/directvolumemigration/pvcs.go

```diff
@@ -72,7 +75,35 @@ func (t *Task) createDestinationPVCs() error {
 	return nil
 }
 
-func (t *Task) getDestinationPVCs() error {
+func (t *Task) getDestinationPVCs() (bool, error) {
 	// Ensure PVCs are bound and not in pending state
-	return nil
+	// Get client for destination
+	destClient, err := t.getDestinationClient()
+	if err != nil {
+		return false, err
+	}
+
+	pvcMap := t.getPVCNamespaceMap()
+	selector := labels.SelectorFromSet(t.buildDVMLabels())
+	for ns, _ := range pvcMap {
+		pvcList := corev1.PersistentVolumeClaimList{}
+		err = destClient.List(
+			context.TODO(),
+			&k8sclient.ListOptions{
+				Namespace:     ns,
+				LabelSelector: selector,
+			},
+			&pvcList)
+		if err != nil {
+			return false, err
+		}
+		for _, pvc := range pvcList.Items {
+			if pvc.Status.Phase == corev1.ClaimLost {
+				t.Log.Info("The PVC in lost state", pvc.Name, pvc.Status.Phase)
```

Comment:
Hmm this seems wrong. We are logging the PVC is in `ClaimLost` state which will break the rsync transfer but returning `true, nil` right?

Reply:
Ideally this is data that is bubbled back up to the user